### PR TITLE
Update Flux post with new apiVersion for operator

### PR DESCRIPTION
**Pull request by MrSimonEmms**
Signed-off-by: Simon Emms <simon@simonemms.com>

## Description

Add notification of new `apiVersion` for OpenFaaS operator

## Motivation and Context

Have run this setup with [a project I'm working on](https://github.com/foundry4/appeal-planning-decision/pull/290) and wanted to share my findings

## Have you applied the editorial and style guide to your post?

See the [README.md](README.md)

## How have you tested the instructions for any tutorial steps?

Yes

## Types of changes

- [ ] New blog post
- [x] Updating an existing blog post
- [ ] Updating part of the page page
- [ ] Adding a new web-page

## Checklist:

- [x] I have given attribution for any images I have used and have permission to use them under Copyright law
- [x] My code follows the [writing-style of the publication](README.md) and I have checked this
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

File: _posts/2020-01-16-openfaas-flux.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ This is a step-by-step guide on how to set up a [GitOps](https://www.weave.works
 workflow for OpenFaaS on Kubernetes with [Flux](https://github.com/fluxcd/helm-operator) and its [Helm Operator](https://github.com/fluxcd/helm-operator).
 You'll be installing OpenFaaS and deploy functions on a Kubernetes cluster in a declarative manner, using only a GitHub repository and the git CLI.
 
+> Update: 2021-01-15
```
Comment (alexellis) [inline _posts/2020-01-16-openfaas-flux.md:20]: Can we just change it in the post?

Comment (MrSimonEmms) [reply _posts/2020-01-16-openfaas-flux.md:20]: Do you know what, when I looked it this yesterday I couldn't see any references for looking. I clearly needed a morning of woodworking and bees to enable me to see things again.

I've updated the GitHub repo behind this post

**Comment by MrSimonEmms** [comment]
@alexellis updated as requested

**Review by alexellis — APPROVED** [review]
Lgtm

**Pull request merged**